source/guides/account/scams-and-security.rst

```diff
@@ -0,0 +1,104 @@
+.. post:: 24 Feb, 2022
+    :category: Account
+    :tags: wallet
+    :excerpt: 1
+    :nocomments:
+
+######################
+Common Hacks and Scams
+######################
+
```

Comment:
I am missing a couple of sentences explaining what this page is about and what you'll learn here.

Reply:
Proposed text: 
"It should come as no surprise that cryptocurrency users are targeted by a wide variety of scammers and thieves. While some attempts may come in familiar forms like impersonation and misinformation, but the digital nature of cryptocurrencies opens the door to a number of novel approaches. This page is intended to help you ensure the security if your assets by explaining a number of common scams and security considerations. This list is by no means an authoritative list of hacks and scams, and you are strongly encouraged to research the topic further for the sake of your wallet's security."

Reply:
Looks  good!  Why don't  you  commit it so we can discuss it more comfortably over the actual changes?
For instance "While... but..." does not sound right, and there's repetition of the word "list".